k-distribution/tests/pyk/defn-tests/build-config.py

```diff
@@ -22,6 +22,11 @@
     defn_name = sys.argv[2][:-5].split("/")[2]
     mod = KFlatModule(defn_name.upper(), ["IMP"], [kast_term])
     kast_term = KDefinition(defn_name, [mod], requires = [KRequire("imp")])
+elif isKClaim(kast_term):
+    kast_term = minimizeRule(kast_term)
```

Comment:
You'll need to modify `minimizeRule` in `k-distribution/src/main/scripts/lib/pyk/kastManip.py` (line 316) as well to handle the `Claim` nodes correctly.